packages/slice-machine/components/ListItem/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import { TextWithTooltip } from "@components/Tooltip/TextWithTooltip";
```

Comment:
this is the .js file converted into .tsx

Reply:
nice